bin/serial-number

```diff
@@ -7,11 +7,17 @@ var fs = require('fs');
 
 var serialNumber = require('../');
 
+var cmdprefix = '/usr/sbin/';
```

Comment:
Let's not change existing behavior by setting a default. This should be an empty string if the user did not choose to specify a custom prefix with the option.